src/main/java/org/tron/core/db/TransactionTrace.java

```diff
@@ -75,9 +88,10 @@ public boolean needVM() {
     return this.trxType == TRX_CONTRACT_CALL_TYPE || this.trxType == TRX_CONTRACT_CREATION_TYPE;
   }
 
+  public long txStartTimeInMs;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make txStartTimeInMs a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1496&issues=AWW-LbF7ashydt7kd_jb&open=AWW-LbF7ashydt7kd_jb)

[ISSUE_KEY]: AWW-LbF7ashydt7kd_jb